pstoremgr/pstoremgr.go

```diff
@@ -243,41 +244,50 @@ func (pm *Manager) LoadPeerstore() (addrs []ma.Multiaddr) {
 
 // SavePeerstore stores a slice of multiaddresses in the peerstore file, one
 // per line.
-func (pm *Manager) SavePeerstore(pinfos []peer.AddrInfo) {
+func (pm *Manager) SavePeerstore(pinfos []peer.AddrInfo) error {
 	if pm.peerstorePath == "" {
-		return
+		err := errors.New("empty peerstore path")
+		logger.Error(err)
+		return err
```

Comment:
No, why? This code is clearly saying that using an empty path in the peerstore is a no-op for Save. Don't change behaviour.